scripts/patch-release/patch-release.go

```diff
@@ -55,45 +57,47 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
+	currentReleaseBranch := branchPrefix + version.Version
+	for _, version := range releaseVersions {
```

Comment:
You need to iterate over the release branches and retrieve the right `version` for them.

Reply:
I don't understand  what  you mean by "right version".
I thought we would have 	releaseMinorVersions = []string{"1.29", "1.28", "1.27"}
then loop through that slice, prepend the release prefix and create the PR.

Reply:
I still don't get it. Currently now, we have 1.30.0  in the internal/version/version.go
but we have 1.29 as the first element of the slice, how does that work?

or do you mean We should take the elements of the slices, using 1.29 as an example, 
then bump up the patch version so it becomes 1.29.1, then make the release PR
"release-1.29.1"

Reply:
> or do you mean We should take the elements of the slices, using 1.29 as an example,
then bump up the patch version so it becomes 1.29.1, then make the release PR
"release-1.29.1"

Exactly. You have to retrieve `1.29.1` by incrementing the latest release on the `release-1.29` branch. That's the tricky part here.